lib/src/utils.rs

```diff
@@ -117,16 +117,12 @@ pub(crate) fn spawn_editor(tmpf: &tempfile::NamedTempFile) -> Result<()> {
     let argv0 = editor_args
         .next()
         .ok_or_else(|| anyhow::anyhow!("Invalid editor: {editor}"))?;
-    let status = Command::new(argv0)
+    Command::new(argv0)
         .args(editor_args)
         .arg(tmpf.path())
         .lifecycle_bind()
-        .status()
-        .context("Spawning editor")?;
-    if !status.success() {
-        anyhow::bail!("Invoking editor: {editor} failed: {status:?}");
-    }
-    Ok(())
+        .run()
```

Comment:
Thanks for putting this is a separate commit with a good message.  I was really confused when I first pulled this up in files view. :)